community-membership.md

```diff
@@ -219,3 +219,33 @@ The following apply to the subproject for which one would be an owner.
 - Ensure a healthy process for discussion and decision making is in place.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
+
+### Resolving technical conflicts within a SIG
+
+From time to time, the Maintainers for a given OpenTelemetry SIG may be unable
+to reach consensus on a technical issue. While it's healthy and appropriate to
+make a sincere attempt to understand all points of view and consider the
+tradeoffs, it's also healthy to occasionally "disagree and commit."
+
+Within OpenTelemetry, SIG Maintainers are chosen specifically _because_ they
+are domain experts, so we would like to keep as much of the decision-making
+authority with the SIG Maintainers rather than immediately "escalating" to the
+overall OpenTelemetry-wide Technical Committee. As such, this is
+OpenTelemetry's process for resolving technical issues where Maintainers cannot
+reach consensus:
+
+1. The SIG Maintainers should succintly document the options under
+consideration as a GitHub issue within the SIG's respective repo (note that it
+is *not* required to document the complete framing and pros/cons – just the
+actual go-forward options themselves).
+2. Each SIG Maintainer must formally vote for their choice by commenting on
+that issue.
+3. The option receiving the most votes wins.
+4. If the vote is split, the OpenTelemetry TC should be brought into the
+discussion, and the TC itself gets a (single) tiebreaking vote.
+
+While inevitably these sorts of decisions will be "disappointing for somebody,"
```

Comment:
No need for quotes here. Quotes could be interpreted to imply that disappointment is invalid.

```suggestion
While inevitably these sorts of decisions will be disappointing for somebody,
```